openslides/core/config.py

```diff
@@ -104,6 +105,29 @@ def __setitem__(self, key, value):
                 if not isinstance(comment['public'], bool):
                     raise ConfigError(_('public property has to be bool.'))
 
+        if config_variable.input_type == 'logo':
+            if not isinstance(value, dict):
+                raise ConfigError(_('logo has to be a dict.'))
+            whitelist = (
+                'path',
+                'display_name',
+            )
+            for required_entry in whitelist:
+                if required_entry not in value:
+                    raise ConfigError(_('{} has to be given.'.format(required_entry)))
+                if not isinstance(value[required_entry], str):
+                    raise ConfigError(_('{} has to be a string.'.format(required_entry)))
+            # TODO: If this is uncommented, I get a 404 not found on the client.
```

Comment:
Please remove this TODO and the code. I don't see a problem of someone saves additional unused properties into the config value.